### PR TITLE
FCOS_RT

### PR DESCRIPTION
**Pull request by miemie2013**
FCOS_RT

### CODE REVIEW

File: ppdet/modeling/backbones/dla.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
+# Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
```
Comment (still-wait) [inline ppdet/modeling/backbones/dla.py:1]: please modify `2019 -> 2020`

**Review by still-wait — COMMENTED** [review]
1.  Need to format the code, you can `pip install pre-commit` and execute `pre-commit` before git commit.
2. Please update mAP in README.md, and do you have a pre-trained model, you can also provide it to us through Baidu Cloud or AIstudio etc...

Thanks for your contribution.

File: ppdet/modeling/backbones/dla.py [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+        out = L.relu(out)
+        return out
+
+
+
+class Root(object):
+    def __init__(self, norm_type, in_channels, out_channels, kernel_size, residual, name=''):
+        super(Root, self).__init__()
+        bn, gn, af = get_norm(norm_type)
+        self.conv = Conv2dUnit(in_channels, out_channels, kernel_size, stride=1, bias_attr=False, bn=bn, gn=gn, af=af, act=None, name=name+'.conv')
```
Comment (heavengate) [inline ppdet/modeling/backbones/dla.py:65]: try `pip install pre-commit` and run `pre-commit run -s HEAD^ -o HEAD` to format code, code not format by `pre-commit` will failed in CI check

File: ppdet/modeling/backbones/dla.py [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+        children = x
+        x = L.concat(list(x), axis=1)
+        x = self.conv(x)
+        if self.residual:
+            x += children[0]
+        x = L.relu(x)
+        return x
+
+
+class Tree(object):
```
Comment (heavengate) [inline ppdet/modeling/backbones/dla.py:78]: Could you kindly give class and argument descriptions in comments

**Pull request closed without merging**